soh/src/code/z_play.c

```diff
@@ -1092,6 +1119,8 @@ void Play_Update(PlayState* play) {
                 // Gameplay stat tracking
                 if (!gSaveContext.sohStats.gameComplete) {
                       gSaveContext.sohStats.playTimer++;
+                      gSaveContext.sohStats.sceneTimer++;
+                      gSaveContext.sohStats.roomTimer++;
```

Comment:
could be cool to refactor to use hooks here, but since we're already using `playTimer` without them that feels out of scope for this PR. probably worth making a new issue for refactoring stats tracking to use hooks.